test/quantization/test_qat.py

```diff
@@ -247,6 +266,16 @@ def test_qat_8da4w_quantizer_disable_fake_quant(self):
             enable_8da4w_fake_quant,
         )
 
+        def assert_fake_quant_enabled(m: torch.nn.Linear, enabled: bool):
+            self.assertTrue(isinstance(m.weight, AffineFakeQuantizedTensor))
+            self.assertEqual(m.weight.fake_quant_enabled, enabled)
+            self.assertTrue(hasattr(m, _QAT_LINEAR_SUBCLASS_INPUT_PREHOOK))
+            (_, handle) = getattr(m, _QAT_LINEAR_SUBCLASS_INPUT_PREHOOK)
```

Comment:
btw, this feels like an implementation detail, so ideally we can check through other ways like check if the gradient of weight is updated

Reply:
yeah I'll add a separate test for that

Reply:
yeah sure, I'll add a separate test for that. I feel it's still good to test that the prehook was installed correctly though